spec/features/admin/export_advisers_list_spec.rb

```diff
@@ -0,0 +1,108 @@
+# Feature: Advisors list export
+#  As the person maintaining the RAD database.
+#  In order to get rid of unwanted advisors.
+#  I need to be able to cross reference a large list (4k) of unauthorised advisers against records in the RAD database.
+#  The list needs to be cleared up so customers do not get sent to unapproved advisers and RAD maintain it's credibility.
+#
+
+## BACKGROUND
+#
+# Firm appearing as St James' Place partners on the FCA register are not
+# eligible to appear on the MAS Directory as they cannot offer a 'whole of
+# market' service on all products.
+#
+# The criteria for entry to the Money Advice Service directory was decided by
+# an independent panel consisting of both consumer and advisers representative
+# bodies.
+# We recently found that some of this advisers have been entering our
+# directory through other company that they are creating. These one are
+# potentialy scams.
+#
+# In order to make sure we don't have any of these advisers in our directory
+# (RAD). Therefore, we need to compare the list of advisers registered under
+# St James place on the FCA register with our database.
```

Comment:
I dont think this is the appropriate place for this information. Your tests are nicely worded and explain what's going on well enough.